RobotCode/src/main/java/frc/robot/Subsystems/ShooterSubsystem.java

```diff
@@ -0,0 +1,34 @@
+package frc.robot.Subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.lib.core.motors.TeamSparkMAX;
+
+public class ShooterSubsystem extends SubsystemBase
+{
+	public TeamSparkMAX shooterMotor, preMotor;
+
+	private static double voltage = 12;
+
+	public ShooterSubsystem()
+	{
+
+		shooterMotor = new TeamSparkMAX("Shooter Motor", 0);
+		preMotor = new TeamSparkMAX("preMotor", 0);
+		shooterMotor.enableVoltageCompensation(voltage);
+		preMotor.enableVoltageCompensation(voltage);
```

Comment:
Need to set neutral modes for the motors, ideally brake for both

Also need ports in the ports class

Reply:
for team spark max, there appears to me no neutral mode

Reply:
Check the documentation, it should be called "idlemode" or something

Reply:
fixed